ws-nextjs-app/fixtures/postFixture.json

```diff
@@ -0,0 +1,503 @@
+{
+  "data": {
```

Comment:
In the Simorgh app we have fixture files for entire pages at the route the repo: https://github.com/bbc/simorgh/tree/latest/data

It might be better to co-locate the fixture data with the story that uses it in the Posts directory? This follows one of our component standards: https://github.com/bbc/simorgh/blob/latest/docs/Coding-Standards/Clean-Code.stories.mdx#place-code-as-close-to-where-its-relevant-as-possible-colocation